config/locales/sv.yml

```diff
@@ -1252,7 +1252,14 @@ sv:
         message_text:
           expire_alert_html: "Ditt medlemsskap gick ut <span class='date-warning'> %{expire_date}.</span>"
           renew_membership: Du kan förnya ditt medlemsskap och fortsätta dra nytta av medlemsfördelarna genom att logga in och betala.
```

Comment:
medlemsskap  = medlemskap

Reply:
oops - I didn't update this one.

Reply:
It can be a separate PR, its a systemic disease